static/src/js/components/ErrorBanner/ErrorBanner.jsx

```diff
@@ -16,28 +17,77 @@ import Row from 'react-bootstrap/Row'
  */
 export const ErrorBanner = ({
   dataTestId,
-  message
-}) => (
-  <Row>
-    <Col>
-      <Alert variant="danger">
-        <Alert.Heading>Sorry!</Alert.Heading>
+  message,
+  previousURL
+}) => {
+  const [hasCMRLagError, setHasCMRLagError] = useState(false)
+  const [conceptIdDoesNotExist, setConceptIdDoesNotExist] = useState(false)
+  const currentURL = window.location.pathname
+  const conceptKeywords = ['/collections/', '/variables/', '/services/', '/tools/']
+
+  // Catches if we are coming from a published record, indicating this is a CMR lag error
+  useEffect(() => {
+    if (previousURL && conceptKeywords.some((keyword) => previousURL.includes(keyword))) {
```

Comment:
See comment below, but you can use the same helper here as well, just pass it the previousURL